src/main/java/ludogorie_soft/reservations_platform_api/dto/PropertyRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package ludogorie_soft.reservations_platform_api.dto;
+
+import lombok.Data;
+
+@Data
+public class PropertyRequestDto {
+
+    private String ownersEmail;
```

Comment:
Is there a reason to use plural form - instead of `ownerEmail`?

Reply:
Field has been changed to ownerEmail